worlds/mmx3/Rules.py

```diff
@@ -0,0 +1,451 @@
+        EventName.toxic_seahorse_rematch
+    ],
+    "Tunnel Rhino": [
+        f"{RegionName.tunnel_rhino_climbing} -> {RegionName.tunnel_rhino_boss}",
+        LocationName.doppler_lab_3_tunnel_rhino,
+        EventName.tunnel_rhino_rematch
+    ],
+    "Volt Catfish": [
+        f"{RegionName.volt_catfish_inside} -> {RegionName.volt_catfish_boss}",
+        LocationName.doppler_lab_3_volt_catfish,
+        EventName.volt_catfish_rematch
+    ],
+    "Crush Crawfish": [
+        f"{RegionName.crush_crawfish_inside} -> {RegionName.crush_crawfish_boss}",
+        LocationName.doppler_lab_3_crush_crawfish,
+        EventName.crush_crawfish_rematch
+    ],
+    "Neon Tiger": [
+        f"{RegionName.neon_tiger_hill} -> {RegionName.neon_tiger_boss}",
+        LocationName.doppler_lab_3_neon_tiger,
+        EventName.neon_tiger_rematch
+    ],
+    "Gravity Beetle": [
+        f"{RegionName.gravity_beetle_inside} -> {RegionName.gravity_beetle_boss}",
+        LocationName.doppler_lab_3_gravity_beetle,
+        EventName.gravity_beetle_rematch,
+    ],
+    "Blast Hornet": [
+        f"{RegionName.blast_hornet_bit_byte} -> {RegionName.blast_hornet_boss}",
+        LocationName.doppler_lab_3_blast_hornet,
+        EventName.blast_hornet_rematch
+    ],
+    "Bit": [
+        LocationName.bit_defeat
+    ],
+    "Byte": [
+        LocationName.byte_defeat
+    ],
+    "Hotareeca": [
+        f"{RegionName.toxic_seahorse_underwater} -> {RegionName.toxic_seahorse_hootareca}"
+    ],
+    "Hell Crusher": [
+        f"{RegionName.tunnel_rhino_wall_jump} -> {RegionName.tunnel_rhino_hell_crusher}",
+    ],
+    "Worm Seeker-R": [
+        f"{RegionName.neon_tiger_start} -> {RegionName.neon_tiger_worm}",
+    ],
+    "Shurikein": [
+        f"{RegionName.blast_hornet_conveyors} -> {RegionName.blast_hornet_shurikein}",
+    ],
+    "Vile": [
+        f"{RegionName.vile_before} -> {RegionName.vile_boss}",
+    ],
+    "Dr. Doppler's Lab 1 Boss": [
+        LocationName.doppler_lab_1_boss,
+        EventName.dr_doppler_lab_1_clear
+    ],
+    "Dr. Doppler's Lab 2 Boss": [],
+    "Doppler": [
+        f"{RegionName.dr_doppler_lab_3_after_rematches} -> {RegionName.dr_doppler_lab_3_boss}",
+        EventName.dr_doppler_lab_3_clear
+    ],
+    "Sigma": [
+        f"{RegionName.dr_doppler_lab_4} -> {RegionName.dr_doppler_lab_4_boss}"
+    ],
+    "Kaiser Sigma": [
+        f"{RegionName.dr_doppler_lab_4} -> {RegionName.dr_doppler_lab_4_boss}"
+    ]
+}
+
+
+def set_rules(world: MMX3World):
+    player = world.player
+    multiworld = world.multiworld
+    jammed_buster = world.options.jammed_buster.value
+
+    multiworld.completion_condition[player] = lambda state: state.has(ItemName.victory, player)
+
+    # Hunter base entrance rules
+    set_rule(multiworld.get_entrance(f"{RegionName.intro_stage} -> {RegionName.blizzard_buffalo}", player),
+             lambda state: state.has(ItemName.stage_blizzard_buffalo, player))
```

Comment:
```suggestion
    set_rule(world.get_entrance(f"{RegionName.intro_stage} -> {RegionName.blizzard_buffalo}"),
             lambda state: state.has(ItemName.stage_blizzard_buffalo, player))
```
There's a newer `world.get_entrance` which doesn't require the player number (since that's in the `world` already)

Reply:
That's useful :o